apps/carbon/src/App.tsx

```diff
@@ -36,7 +36,7 @@ const extensions = [
 const App = (props: AppProps) => (
   <ErrorBoundary>
     <AppShellProvider theme={`${props.theme ? props.theme : "theme-dark"}`}>
-      <AppShell embedded>
+      <AppShell embedded fullWidthContent={undefined} sideNavigation={undefined} topNavigation={undefined}>
```

Comment:
This looks strange to me. I suspect it might be related to the type definitions in the ui-components. Shouldn’t those types be optional?

Reply:
Totally, right I did it only because when we start exporting types the app started failed, so I thought it would be a part of diferent PR to normalize types and havea. ticket for that.